extensions/aris/src/extension.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import * as vscode from 'vscode';
+import { AppContext } from './appContext';
+import { registerTreeDataProvider, ControllerTreeDataProvider } from './bigDataCluster/tree/controllerTreeDataProvider';
+import { registerCommands } from './bigDataCluster/commands';
+import { IconPath } from './bigDataCluster/constants';
+
+export function activate(extensionContext: vscode.ExtensionContext) {
+	IconPath.setExtensionContext(extensionContext);
+	let appContext = new AppContext(extensionContext);
+	let treeDataProvider = new ControllerTreeDataProvider();
```

Comment:
Let's not write another controller class. This pattern is only useful if we're unit testing these, which we're not. Please put the command and tree registration directly in here, and reference functions that are testable from here.

Reply:
fixed.